templates/managed/index.html

```diff
@@ -154,11 +157,11 @@ <h1>Managed IT Services you can rely on</h1>
 </section>
 
 <section class="u-fixed-width">
-  <hr class="p-rule" />
+  <hr class="p-rule">
```

Comment:
@lyubomir-popov according to our convention all HTML tags should be self-closed.